### DISCUSSION TITLE
Question about Changing Theme Colors (change the config file but changes didn't apply)

### DISCUSSION THREAD

**Discussion by Sacolevas** [tsujan/Kvantum#598] [Q&A]
Hello!

I have a question, which I don't know if could be described as an issue or it is due to my lack of experience using Kvantum. 

I am using the KvYaru theme, but didn't like the orange color in the highlights, for example. Wanted to change it to a color palette more consistent with the Mint-Y theme I use in Linux Mint. 

Opening the Kvantum Manager, it says:

"These are the settings that can be safely changed.
For the others, edit this file:
/home/... /.config/Kvantum/KvYaru#/KvYaru#.kvconfig"

So, I saved the configuration, then edited the "KvYaru#.kvconfig" file removing all ocurrences of orange and replacing them with the green colors from Linux Mint. Saved the file, reboot the computer, entered again and nothing has changed, the orange colour are still there. 

What should I do? Why changes are not being applied? Thanks in advance

**Comment by tsujan** [comment]
I moved it to Discussions → Q&A because similar questions have been asked several times.

Kvantum's *background* colors aren't provided only by the config file but also by the SVG image because Kvantum is an SVG style engine.

Unlike simple style engines, Kvantum can give complex backgrounds to various widgets. Those backgrounds are in the SVG file and can be much more complex than a mere color. The toggled buttons of the default theme or those of `KvSimplicity` are some examples.

However, the background colors defined in the config file are necessary because they may be used in special cases. They should be good enough.

In contrast, the *foreground* colors are always the colors that are defined in the config file. They're used to draw texts and some other elements.

**Comment by Sacolevas** [comment]
Oh I see, thanks. So I cannot change anything in this theme even if I edit the config files, right?
By any chance Is there a Kvantum there that resemble the green of Linux Mint Y-Theme for the sake of consistency (and I like the mint green colour)? 

I love the way KVYaru looks, except the orange color. Of course, it is not a deal breaker, i.e, I will continue using Kvantum, but I will be grateful if you know of some alternative inside Kvantum. 

Thanks a lot!

**Reply by tsujan** [reply]
> So I cannot change anything in this theme

Of course , you can. Everything you might need is explained in the two PDF files inside the source. You need to make another theme based on KvYaru. I've seen many users doing so.

KVYaru is made based on the GTK theme Yaru; so it should follow Yaru.
